pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -329,6 +330,10 @@ func TestPriorityQueue_Update(t *testing.T) {
 	}
 	updatedPod := medPriorityPodInfo.Pod.DeepCopy()
 	updatedPod.ClusterName = "test"
+
+	// Move clock by podInitialBackoffDuration, so that pods in the unschedulableQ would pass the backing off,
```

Comment:
You should unit test both cases. So do one scenario where the pod goes to backoff queue, and one where it goes directly to active.

We also need a test for the pod is in backoff queue and stays there after an update.